dbms/src/Core/SpillHandler.cpp

```diff
@@ -64,21 +64,21 @@ void SpillHandler::spillBlocks(const Blocks & blocks)
     {
         Stopwatch watch;
         RUNTIME_CHECK_MSG(spiller->spill_finished == false, "{}: spill after the spiller is finished.", spiller->config.spill_id);
-        LOG_INFO(spiller->logger, "Spilling {} blocks data into temporary file {}", blocks.size(), current_spill_file_name);
-        size_t spilled_data_size = 0;
+        auto block_size = blocks.size();
+        LOG_INFO(spiller->logger, "Spilling {} blocks data into temporary file {}", block_size, current_spill_file_name);
+        size_t total_rows = 0;
         if (unlikely(writer == nullptr))
         {
             writer = std::make_unique<SpillWriter>(spiller->config.file_provider, current_spill_file_name, blocks[0].cloneEmpty(), spiller->spill_version);
         }
         for (const auto & block : blocks)
         {
-            auto block_bytes_size = block.bytes();
+            total_rows += block.rows();
             writer->write(block);
-            spilled_data_size += block_bytes_size;
         }
         double cost = watch.elapsedSeconds();
         time_cost += cost;
-        LOG_INFO(spiller->logger, "Finish Spilling data into temporary file {}, spilled data size: {}, time cost: {:.3f} sec.", current_spill_file_name, spilled_data_size, cost);
+        LOG_INFO(spiller->logger, "Spilled {} rows from {} blocks into temporary file, time cost: {:.3f} sec.", total_rows, block_size, cost);
```

Comment:
Use rows instead of `spilled_data_size` because [`block.bytes()` is highly overestimated if there contains `ColumnAggregateFunction`](https://github.com/pingcap/tiflash/blob/0bb9cb4819b458b49799227f00441e3e74e98488/dbms/src/Columns/ColumnAggregateFunction.cpp#L225), it will mislead the user/developer